pkgs/by-name/ru/rustdesk-flutter/package.nix

```diff
@@ -163,8 +163,11 @@ flutter319.buildFlutterApplication rec {
   patches = [ ./make-build-reproducible.patch ];
 
   postPatch = ''
+    if [ $cargoDepsCopy ]; then # That will be inherited to buildDartPackage and it doesn't have cargoDepsCopy
```

Comment:
Can you explain why this fixes the issue? :D Because I don't understand it.

Also, the `substituteInPlace` should now be indented one level imo to keep consistency.